vars/withMavenSettings.groovy

```diff
@@ -15,8 +15,9 @@
  * @param nexusCredentials Jenkins credentials which provide USERNAME and PASSWORD to an account which enables Nexus interaction
  * @param cesFQDN the full qualified domain name of the current CES instance, f. i. <code>cesinstance.stage.host.tld</code>
  * @param pathToLocalMavenRepository without the .m2 directory part, f. i. <code>/usr/share/maven</code>. The suffix <code>/.m2/repository</code> will be added automatically.
+ * @param mirrorNexusPath relativ path to a maven central mirror hosted inside a nexus instance. The suffix <code>/nexus/repository</code> will be added automatically.
```

Comment:
- typo: `relativ` -> e
- provide example for generated output string: https.../../m.n.p.